src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -1098,13 +1098,12 @@ private stmt_ ::= expr_stmt
 // Utils
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 
-GTGTEQ ::= '>' '>' '='
-GTGT   ::= '>' '>'
-GTEQ   ::= '>' '='
-
-LTLTEQ ::= '<' '<' '='
-LTLT   ::= '<' '<'
-LTEQ   ::= '<' '='
+external gtgteq ::= collapse GTGTEQ GT GT EQ
```

Comment:
What do you think about using `'>>='`  tokens here? I think they are a bit more readable then named counterparts (you  will need to add `GTGTEQ = ">>="` to the tokens array at the beginning of the file).

Overall it is more typing, but also more readability.